script/generate_helm.sh

```diff
@@ -21,7 +21,7 @@ then
     GO111MODULE=on go get github.com/norwoodj/helm-docs/cmd/helm-docs
 fi
 
-~/go/bin/helm-docs -t ${DIR}/../helm/README.md.gotmpl ${DIR}/../helm/
+${GOPATH}/bin/helm-docs -t ${DIR}/../helm/README.md.gotmpl ${DIR}/../helm/
```

Comment:
I ran into issues while running `make helm` as my GOPATH is somewhere else, so I bundled this change which seems like an improvement but let me know if you'd prefer this be reverted or put in its own PR.